src/scripts/ci/publish.ts

```diff
@@ -939,6 +961,10 @@ async function publishPackages(
     for (const pkgName of currentBatch) {
       const pkg = packages[pkgName]
 
+      if (pkg.name === '@prisma/tests') {
+        continue
```

Comment:
I wanted to ask the question "Are we publishing this pkg to npm?" I guess that answers it by a no ✅ Perfect!